make/golang.mk

```diff
@@ -1,7 +1,7 @@
 # This is the default Clever Golang Makefile.
 # It is stored in the dev-handbook repo, github.com/Clever/dev-handbook
 # Please do not alter this file directly.
-GOLANG_MK_VERSION := 0.2.0
+GOLANG_MK_VERSION := 0.2.1
```

Comment:
shouldn't this be minor to convey the breaking nature of it?

Reply:
👍 a0140b7